pkg/ibmvpc/examples/out/lint_out/acl3_shadowed_rules_Lint

```diff
@@ -0,0 +1,55 @@
+network ACL demilune-humorless-captain-lurex of VPC test-vpc1-ky has no resources attached to it
+security group barbecue-frayed-varied-average of VPC test-vpc1-ky has no resources attached to it
+________________________________________________________________________________________________________________________________________________________________________________________________________
+
+"Blocked TCP response" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In the connection from test-vpc1-ky/db-endpoint-gateway-ky[10.240.30.7] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP response is blocked
+In the connection from test-vpc1-ky/vsi2-ky[10.240.20.4] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP response is blocked
+In the connection from test-vpc1-ky/vsi3a-ky[10.240.30.5] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP response is blocked
+In the connection from test-vpc1-ky/vsi3b-ky[10.240.30.6] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP response is blocked
+In the connection from test-vpc1-ky/vsi3c-ky[10.240.30.4] to test-vpc1-ky/vsi1-ky[10.240.10.4] TCP response is blocked
+________________________________________________________________________________________________________________________________________________________________________________________________________
+
+"rules of network ACLs implying different connectivity for different endpoints within a subnet" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In VPC test-vpc1-ky, network ACL acl2-ky rule's indexed 6 splits subnet subnet2-ky (10.240.20.0/24). Splitting rule details: index: 6, direction: outbound , src: 10.240.20.0/28 , dst: 10.240.10.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 1 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 1, direction: outbound , src: 10.240.30.0/31 , dst: 10.240.20.0/24, conn: all, action: allow
+In VPC test-vpc1-ky, network ACL acl3-ky rule's indexed 3 splits subnet subnet3-ky (10.240.30.0/24). Splitting rule details: index: 3, direction: inbound , src: 10.240.20.0/24 , dst: 10.240.30.0/31, conn: all, action: allow
+________________________________________________________________________________________________________________________________________________________________________________________________________
+
+"rules of network ACLs that are shadowed by higher priority rules" issues:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In VPC test-vpc1-ky network ACL acl2-ky's rule 6 is redundant. It is shadowed by higher priority rules
+	Rule's details: index: 6, direction: outbound , src: 10.240.20.0/28 , dst: 10.240.10.0/24, conn: all, action: allow
+		Shadowing rules:
+		index: 1, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: protocol: icmp, action: allow
+		index: 2, direction: outbound , src: 10.240.20.0/24 , dst: 10.240.10.0/24, conn: all, action: allow
```

Comment:
Yes, I think we do.
It will require a change in the definition of `NACL` redundancy, and potentially require separating the `SG` and `NACL` computation. I'll document it in the code.

Reply:
In 2nd thought... I am not so sure about the `NACL` - specifically given that the term we use is "shadowed by".
Perhaps for `NACL `we should leave it as is.
A rule is shadowed in `NACL `if the rule will never determine the actual allow/deny status (of any connection in any configuration)
We can add another checker for the additional notion of redundancy in `NACL`, the one that also considers lower priority rules. We can do it for this release if important. I will also change the name in `NACL` to have shadow in it.